content/post/ddos-detection.md

```diff
@@ -0,0 +1,413 @@
++++
+title = "DDoS Attack Detection with Wallaroo: A Real-time Time Series Analysis Example"
+draft = false
+date = 2017-11-24T00:00:00Z
+tags = [
+    "wallaroo",
+    "windowing",
+    "partitioning",
+    "python",
+    "example",
+    "tutorial",
+    "ddos",
+    "timeseries",
+    "anomaly detection"
+]
+categories = [
+    "Windowing",
+    "Examples"
+]
+description = "Building a DDoS attack detector using timeseries analysis in real time with Wallaroo"
+author = "nisanharamati"
++++
+
+
+# DDoS Attack Detection with Wallaroo: A Real-time Time Series Analysis Example
+
+In this post, I would like to demonstrate a real-world use case for Wallaroo: DDoS attack detection in real time, using a stream of request logs from a large group of servers. This example builds on the concepts introduced in [Non-native event-driven windowing in Wallaroo][windowing in wallaroo], adding partitioning, parallelization, and the application of statistical analysis to perform a simplified anomaly detection based on weighted mean and standard deviation to determine when a server is experiencing an uncharacteristic and heavy load.
```

Comment:
this intro presumes i know what Wallaroo is. I think that is a very poor assumption.

Give me a short intro of a couple of sentences that introduces what Wallaroo is.

Also this first paragraph is basically telling me "go read this other post first". so, do i need to do that? i personally would go read the other post first then honestly, probably not come back to this one.

"builds on" to me means "is a prerequisite for understanding this post"

given this isn't a series of part 1, part 2, if prereq is true, that is problematic.